wrapper.ts

```diff
@@ -55,17 +55,38 @@ export function setPlayerMetaData(engine: Engine, player: number, metaData: { na
   return engine;
 }
 
-export function setPlayerSettings(engine: Engine, player: number, settings: { autoCharge?: string }) {
+export function setPlayerSettings(
+  engine: Engine,
+  player: number,
+  settings: {
+    autoCharge?: string;
+    autoIncome?: string;
+    autoBrainstone?: string;
+    itarsAutoChargeToArea3?: string;
```

Comment:
Those can be boolean (checkbox). It's just there's no support for numbers, but there's support for booleans or string enums.